src/components/Footer.jsx

```diff
@@ -132,7 +132,7 @@ export default class Footer extends Component {
 					</a>
 					<a className={'footer__link'} href={'https://marketplace.atlassian.com/apps/1218380/zenuml-sequence-diagram'} target="_blank">&nbsp;Confluence Plugin&nbsp;</a>
 					<a className={'footer__link'} href={'https://chrome.google.com/webstore/detail/web-sequence/kcpganeflmhffnlofpdmcjklmdpbbmef'} target='_blank'>&nbsp;Chrome Extension&nbsp;</a>
-					<a className={'footer__link'} href={'/privacy-policy/index.html'} target='_blank'>&nbsp;Privacy Policy&nbsp;</a>
+					<a className={'footer__link'} href={'privacy-policy/index.html'} target='_blank'>&nbsp;Privacy Policy&nbsp;</a>
```

Comment:
@Ivylyu   link of EULA has been removed because we need to add other links. 
please add the link back to footer